### PR TITLE
Resolve #1805 Cache EthContract on client

### LINKED ISSUE #1805: Cache EthContract on client

### PR DESCRIPTION
**Pull request by InVincible2016**

### CODE REVIEW

File: src/configureStore.js [hunk 1]
```diff
@@ -90,12 +90,25 @@ const serializeTransform = createTransform(
   { whitelist: ['txControllerReducer'] }
 )
 
+const EthContractsTransform = createTransform(
```
Comment (timothywangdev) [inline src/configureStore.js:93]: this doesn't seem to be doing any real transformation

File: src/App.js [hunk 2]
```diff
@@ -327,6 +332,8 @@ class App extends Component {
           store.dispatch(refreshAccessToken())
         }, 1000 * 60 * 50)
       }
+    } else {
+      await store.dispatch(preLoginActions())
```
Comment (timothywangdev) [inline src/App.js:336]: don't we need to fetch ethContract for authenticated users?

File: src/configureStore.js (hunk 1 above)
Comment (InVincible2016) [reply src/configureStore.js:93]: It only picks out the ethContracts from accountReducer.

File: src/App.js (hunk 2 above)
Comment (InVincible2016) [reply src/App.js:336]: It's done inside the postLoginPreparation function for authenticated users

**Pull request merged**